src/cable.cr

```diff
@@ -31,18 +31,19 @@ module Cable
     setting token : String = "token", example: "token"
     setting url : String = ENV.fetch("REDIS_URL", "redis://localhost:6379"), example: "redis://localhost:6379"
     setting disable_sec_websocket_protocol_header : Bool = false
+    setting backend_class : Cable::BackendCore.class = Cable::RedisBackend, example: "Cable::RedisBackend"
+    setting redis_ping_interval : Time::Span = 15.seconds
+    setting restart_error_allowance : Int32 = 20
+    setting on_error : Proc(Exception, String, Nil) = ->(exception : Exception, message : String) do
+      Cable::Logger.error(exception: exception) { message }
+    end
 
     # DEPRECATED
     # only use if you are using stefanwille/crystal-redis
     # AND you want to use the connection pool
     setting pool_redis_publish : Bool = false
     setting redis_pool_size : Int32 = 5
     setting redis_pool_timeout : Float64 = 5.0
-    setting redis_ping_interval : Time::Span = 15.seconds
-    setting restart_error_allowance : Int32 = 20
-    setting on_error : Proc(Exception, String, Nil) = ->(exception : Exception, message : String) do
-      Cable::Logger.error(exception: exception) { message }
-    end
```

Comment:
I just moved these up so we can group which settings are deprecated and which are still in use.